test/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/VmrSyncRepoChangesTest.cs

```diff
@@ -57,41 +57,6 @@ public async Task FileChangesAreSyncedTest()
         await GitOperations.CheckAllIsCommitted(VmrPath);
     }
 
-    [Test]
-    public async Task PackageVersionIsUpdatedOnlyTest()
```

Comment:
We had this functionality so that it's possible to flow a new build of the same SHA. This was a requirement of the SourceBuild team. In those cases, we would only update the version files and be done.
I think this should still be supported.